examples/default/README.md

```diff
@@ -1,8 +1,9 @@
 # Default example
 
-An end-to-end example that uses the module's default variable values.
-This example uses the IBM Cloud terraform provider to:
- - Create a new resource group if one is not passed in.
- - Create a new VPC in the resource group and region provided.
+A default example that uses the module's default variable values.
+This example uses the IBM Cloud Terraform provider to create this infrastructure:
+- A resource group, if one is not passed in.
+- A VPC in the resource group.
+- Security group rules in the default security group in the VPC.
```

Comment:
The code is not touching the default sg in the VPC however - it is adding sg rules to a security group passed as input, or create a new sg